builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -71,32 +72,55 @@ func testAccCheckAWSDBInstanceDestroy(s *terraform.State) error {
 	conn := testAccProvider.Meta().(*AWSClient).rdsconn
 
 	for _, rs := range s.RootModule().Resources {
-		if rs.Type != "aws_db_instance" {
-			continue
-		}
+		if rs.Type == "aws_db_instance" {
```

Comment:
can you help me understand why these changes were made? Why are we now checking `aws_db_subnet_group` ?